docs/design/design_v1.9.md

```diff
@@ -69,196 +67,246 @@ This means we aim to standardize:
 
 `kubeadm init` internal workflow consists of a sequence of atomic work tasks to perform.
 
-The `kubeadm phase` command, (in v1.8 staged under `kubeadm alpha phase`), allows users to invoke individually each task, and ultimately offers a reusable and composable API/toolbox that can be used by other kubernetes bootstrap tools / by any IT automation tool / by advanced user for creating custom clusters. 
+The `kubeadm alpha phase` command allows users to invoke individually each task, and ultimately offers a reusable and composable API/toolbox that can be used by other Kubernetes bootstrap tools / by any IT automation tool / by advanced user for creating custom clusters. 
 
 ### Preflight checks
 
 `kubeadm` executes a set of preflight checks before starting the init, with the aim to verify preconditions and avoid common cluster startup problems. 
-In any case the user can skip preflight checks with the `--skip-preflight-checks` option.
-
-- [warning] If the Kubernetes version to use (passed with the `--kubernetes-version` flag) is one minor version higher than the kubeadm CLI version.
-- Kubernetes system requirements, 
-  - [error] if not linux, 
-  - [error] if not Kernel 3.10+ or 4+ with specific KernelSpec,
-  - [error] if required cgroups subsystem aren't in set up,
-  - [error/warning] if Docker endpoint does not exists or does not work, if docker version v1.11.2 <= x <= v1.13.1,
-- [error] if user is not root,
-- [warning] if kubelet service does not exists, if it is disabled,
-- [warning/error] if Docker service does not  exists, if it is disabled, if it is not active,
-- [warning] if firewalld is active,
-- [error] if API.BindPort or ports 10250/10251/10252 are used,
-- [warning] if connection to https://API.AdvertiseAddress:API.BindPort goes thought proxy,
-- [Error] if /etc/kubernetes/manifest folder already exists and it is not empty,
-- [Error] if /var/lib/kubelet folder already exists and it is not empty,
-- [Error] if/proc/sys/net/bridge/bridge-nf-call-iptables file does not exists/does not contains 1
-- [Error] if swap is off,
-- [Error] if "ip", "iptables",  "mount", "nsenter" commands are not present in the command path,
-- [warning] if "ebtables", "ethtool", "socat", "tc", "touch" commands are not present in the command path,
-- [warning] if extra arg flags for APIServer, ControllerManager,  Scheduler contains some invalid options
-- if external etcd is provided, [Error] if etcd version less than 3.0.14
-- if external etcd is not provided, [Error] if ports 2379 is used, if Etcd.DataDir folder already exists and it is not empty,
-- if authorization mode is ABAC, [Error] if abac_policy.json does not exists
-- if authorization mode is WebHook, [Error] if webhook_authz.conf does not exists
+In any case the user can skip specific preflight checks (or eventually all preflight checks) with the `--ignore-checks-errors` option.
```

Comment:
We now renamed this to `--ignore-preflight-errors` I think, please double check what's at HEAD please